app/src/main/kotlin/org/equeim/tremotesf/ui/addtorrent/AddTorrentFileFragment.kt

```diff
@@ -371,7 +380,9 @@ class AddTorrentFileFragment : AddTorrentFragment(
                     savedInstanceState
                 )
 
-                startDownloadingCheckBox.isChecked = GlobalRpc.serverSettings.startAddedTorrents
+                GlobalRpc.isConnected.launchAndCollectWhenStarted(viewLifecycleOwner) {
+                    startDownloadingCheckBox.isChecked = GlobalRpc.serverSettings.startAddedTorrents
```

Comment:
Will it be reset if user toggles checkbox and e.g. changes device orientation?

Reply:
Yes, dcc09776341ff55819601baaa56e1cbd0233c04f addresses this issue.